pkg/test/test.go

```diff
@@ -15,9 +15,9 @@ type mockComponent interface {
 }
 
 type mockSettings struct {
+	Host        string
 	Debug       bool          `cfg:"debug"`
 	Component   string        `cfg:"component"`
-	Host        string        `cfg:"host"`
```

Comment:
why not keep it configurable as we did for the ports?

Reply:
For the port you can set 0 to indicate that you want to use dynamic ports.
We cannot do the same for the host. So we just remove it from the configuration.

Reply:
could it make sense to use this as the configurable ci host/ip?